airflow_provider_azure_machinelearning/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.0.1b0"
+__version__ = "0.0.1b2"
```

Comment:
Any reason, why we are skipping the release version: 1b1?

Reply:
https://pypi.org/project/airflow-provider-azure-machinelearning/#history

b1 is already published. It was published ealier when I was testing this PR but with PUSH triggering instead of MERGE triggering.